src/webserver/src/php_amule_lib_standalone.cpp

```diff
@@ -608,7 +603,7 @@ int main(int argc, char *argv[])
 
 	//phpdebug = 0;
 
-	CPhpFilter php_filter((CWebServerBase*)0, (CSession *)0,filename, &buffer);
+	php_filter((CWebServerBase*)0, (CSession *)0,filename, &buffer);
```

Comment:
This clearly causes a compilation error, I can't imagine how it worked for you.

Reply:
no, there is no compilation error

In the past with cppcheck 2.1 I saw this warning:

```
src/webserver/src/php_amule_lib_standalone.cpp:611:23: style: Variable 'php_filter' is assigned a value that is never used. [unreadVariable]
 CPhpFilter php_filter((CWebServerBase*)0, (CSession *)0,filename, &buffer);
                      ^
```

but now with cppcheck 2.2 the warning is gone, so, I will revert this change